### PR TITLE
chore: add backend version name length validation

### PR DESCRIPTION
**Pull request by mikebarr24**
Addresses #4787

### CODE REVIEW

**Comment by gsmithun4** [comment]
please remove this validations since we have added the same in DTO level
https://github.com/ToolJet/ToolJet/blob/develop/server/src/services/apps.service.ts#L231

File: server/src/dto/version-create.dto.ts [hunk 1]
```diff
@@ -1,11 +1,12 @@
-import { IsNotEmpty, IsOptional, IsString, IsUUID } from 'class-validator';
+import { IsNotEmpty, IsOptional, IsString, IsUUID, Length } from 'class-validator';
 import { Transform } from 'class-transformer';
 import { sanitizeInput } from '../helpers/utils.helper';
 
 export class VersionCreateDto {
   @IsString()
   @Transform(({ value }) => sanitizeInput(value))
   @IsNotEmpty()
+  @Length(1, 25)
```
[the remark this answers is not in the document]
Comment (mikebarr24) [reply server/src/dto/version-create.dto.ts:9]: Would it be worth adding MinLength as well and then removing
```
    if (!versionName) {
      throw new BadRequestException('Version name cannot be empty.');
    }
```
from apps.service.ts?

[the remark this answers is not in the document]
Comment (mikebarr24) [reply server/src/dto/version-create.dto.ts]: I hope this is OK. I've added the MinLength as well and removed the relevant logic from app.service.ts

File: server/entrypoint.sh [hunk 2]
```diff
@@ -1,18 +0,0 @@
-#!/bin/bash
```
Comment (gsmithun4) [inline server/entrypoint.sh:1]: Looks like its not related to this PR, please revert

**Review by gsmithun4 — CHANGES_REQUESTED** [review]

File: server/entrypoint.sh (hunk 2 above)
Comment (mikebarr24) [reply server/entrypoint.sh]: Sorry! Still a bit clumsy with git. I think I've fixed it here.

Comment (mikebarr24) [reply server/entrypoint.sh]: I wonder would it be better to create a new PR?

Comment (gsmithun4) [reply server/entrypoint.sh]: looks good now

**Review by mikebarr24 — COMMENTED** [review]
I think I've fixed this. Not entirely sure of the review process here so I might have done something wrong.

**Review by mikebarr24 — COMMENTED** [review]
I think I've fixed this. (Sorry! Playing around with github and realise I can't delete this)

**Comment by gsmithun4** [comment]
@mikebarr24 Please resolve conflicts

**Review by gsmithun4 — APPROVED** [review]

**Pull request merged**